spec/models/events/review_comment_spec.rb

```diff
@@ -73,27 +84,33 @@
     end
 
     describe '#created' do
+      before { subject.payload['action'] = 'created' }
+
       it 'sets body' do
         expect {
-          subject.send(:created)
+          subject.resolve
         }.to change { subject.reload.body }.from(nil).to(payload['comment']['body'])
```

Comment:
@hdamico Would it make sense to define the following expressions

```
      let(:payload_comment) { payload.comment }
      let(:payload_comment_body) { payload_comment.body }
```

and change the occurrences of

`payload['comment']['body']`

to

`payload_comment_body` ?

I think it might improve a bit the test ligibility.